xml/System.Net.Security/SslStream.xml

```diff
@@ -41,7 +41,10 @@
     <summary>Provides a stream used for client-server communication that uses the Secure Socket Layer (SSL) security protocol to authenticate the server and optionally the client.</summary>
     <remarks>
       <format type="text/markdown"><![CDATA[  
-  
+
+> [!NOTE]
+>  Be sure you have the private key included in the certificate; cause the server needs it to decrypt the received message otherwise you will get a decryption error on the server-side.
```

Comment:
```suggestion
> Be sure to include the private key in the certificate. The server needs the private key to decrypt the received message. If you omit it, you'll get a decryption error on the server side.
```

Reply:
While this is true in general, Is this remark attached to some specific property? 
Also I think the "message" should be plural. Perhaps something like `... decrypt the received messages and complete handshake.`
However, AFAIK we will not get that far and SslStream will throw PNSP or Authentication Exception if X509Certificate without key is provided. That should be documented on the property it self and delegates returning certificates.
That being said, there is also some fixup code to search through certificate stores and it can work even if given certificate instance does not have a key. But it may be better to guide users to use X509Certificate2 properly.

Reply:
Thanks for your suggestion. Any description to clarify the sample and its related issues is welcome.
Please, commit your suggestions. I'll accept it.

Reply:
How to using the certificates is not the subject of this context. And I have avoided describing it more here. Nonetheless, if you have useful links we can add it here as a reference.